core/chains/evm/logpoller/log_poller.go

```diff
@@ -414,7 +413,7 @@ func (lp *logPoller) HealthReport() map[string]error {
 }
 
 func (lp *logPoller) GetReplayFromBlock(ctx context.Context, requested int64) (int64, error) {
-	lastProcessed, err := lp.orm.SelectLatestBlock(pg.WithParentCtx(ctx))
+	lastProcessed, err := lp.orm.SelectLatestBlock(lp.ctx)
```

Comment:
Why change contexts here?
```suggestion
	lastProcessed, err := lp.orm.SelectLatestBlock(ctx)
```